docs/ChainOrderTransitivity.md

```diff
@@ -0,0 +1,153 @@
+# Transitivity of the chain order
+
+## Context
+
+The abstract Consensus layer currently implements the chain order like this:
+From every header, one can extract a `SelectView`, which has an `Ord` instance.
+A chain `C` is _preferable_ to a chain `C'` if `selectView(C) > selectView(C')` where `C` and `C'` are the tip headers of `C` and `C'`, respectively.
+
+## Examples for intransitive orders
+
+### Current chain order is not transitive
+
+For all Shelley-based eras, we currently use a lexicographic combination of the following comparisons in descending order:
```

Comment:
Part of me doubts whether it's fair to call it lexicographic with the conditional in there.

It's seems fairer to say "we either use lexicographic (length, issue number, VRF) or lexicographic (length, VRF)", eg. I dunno, maybe I'm splitting hairs.

Reply:
I will rephrase and only allude to lexicographic combination :+1: